msi/GitChooseEnvironmentDlg.wxs

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
So, while developing and testing this installer, it is getting tedious clicking through all these dialogs; having said that, people aren't installing GfW multiple times a day. Anyway, I think your point is that the default options are sensible enough (and for any well thought out application and installer, they should be) that, why make the user click through all these dialogs? I don't disagree 😉 .

WRT your last comment, I'm not sure I follow your implication as to it's effect on the current development of this installer. Are you suggesting that the advanced dialog set could be simplified?